abstract_specification.md

```diff
@@ -283,7 +283,7 @@ Parameters:
 
 - uri - location at which to create the object
 - schema - an Arrow Schema defining the per-column schema.
-- platform_config - optional storage-engine specific configuration
+- [platform_config](#platform-configuration) - optional storage-engine specific configuration
```

Comment:
I believe this link is incorrect, and should be linked to the anchor `platform-specific-configuration`.  The same typo is made throughout the doc.

Reply:
Whoops! Leftover from an earlier revision of the text.